src/main/java/com/ait/lienzo/client/core/shape/wires/layout/size/SizeConstraints.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ait.lienzo.client.core.shape.wires.layout.size;
+
+public class SizeConstraints
+{
+    public enum Type
+    {
+        PERCENTAGE, RAW
```

Comment:
just minor comment -> `RAW` means some real value, which finally will be used as pixel units, is it? If so, what about renaming from `RAW` to `PIXEL` or `PX`... WDYT?